stories/layout.stories.js

```diff
@@ -42,6 +46,42 @@ let HighlightDemo = withStateLens({ filter: '' })(
   ))
 )
 
+let MockTagsInput = ({ initialTags, deriveStateFromProps = false }) => {
+  let [tags, setTags] = useState(initialTags)
+  if (deriveStateFromProps && initialTags !== tags) setTags(initialTags)
+  return (
+    <TagsInput
+      tags={tags}
+      addTag={t => setTags(_.concat(tags, t))}
+      removeTag={t => setTags(_.pull(t, tags))}
+      tagStyle={{ color: 'white', backgroundColor: '#0076de', padding: 5 }}
+      PopoverContents={({ tag }) => <div>{tag}</div>}
+    />
+  )
+}
+
+let DynamicProps = ({ Component, propList }) => {
```

Comment:
This is a little hard to follow - why do we need a demo with setInterval?

Reply:
The idea was to test if the input lost focus on rerenders. It kept it in the story but it didn't on our main product, so it's ok to remove.